packages/tooltip/src/components/popover/Popover.tsx

```diff
@@ -45,7 +45,6 @@ type TriggerProp = PopoverTrigger | Array<PopoverTrigger>;
 
 export interface PopoverProps {
   children?: PopoverChildren;
```

Comment:
If we no longer have onRequestClose, there is no reason to use PopoverChildren here, just use `extends PropsWithChildren`